ql/termstructures/volatility/swaption/sabrswaptionvolatilitycube.hpp

```diff
@@ -1151,16 +1157,30 @@ namespace QuantLib {
         return result;
     }
 
+    /*! \deprecated Use XabrSwaptionVolatilityCube instead.
+                    Deprecated in version 1.29.
+    */    
+
     //======================================================================//
-    //                      SwaptionVolCube1 (Sabr)                         //
+    //                      SabrSwaptionVolatilityCube (Sabr)                         //
     //======================================================================//
 
+    //! Swation Volatility Cube SABR 
+    /*! This struct defines two types of SABR Volatility cubes, one for the
+        SABRInterpolation and another one for smiles: SabrSmileSection.
+    */
     struct SwaptionVolCubeSabrModel {
         typedef SABRInterpolation Interpolation;
         typedef SabrSmileSection SmileSection;
     };
 
-    typedef SwaptionVolCube1x<SwaptionVolCubeSabrModel> SwaptionVolCube1;
+
+    typedef XabrSwaptionVolatilityCube<SwaptionVolCubeSabrModel> SabrSwaptionVolatilityCube;
+
+    /*! \deprecated Use SabrSwaptionVolatilityCube instead.
+                    Deprecated in version 1.29.
+    */
+    
```

Comment:
You're missing the actual deprecated typedef here (`SabrSwaptionVolatilityCube` to `SwaptionVolCube1`)